app/presenters/cancellation_presenter.rb

```diff
@@ -4,9 +4,10 @@ class CancellationPresenter < FailurePresenter
 
   attr_reader :referer
 
-  def initialize(referer:)
+  def initialize(referer:, url_options: )
```

Comment:
nit, remove the extra space

```suggestion
  def initialize(referer:, url_options:)
```